jenkins_wrapper_cookbook/Vagrantfile

```diff
@@ -47,6 +47,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     chef.data_bags_path = 'data_bags'
     chef.environments_path = 'environments'
     chef.nodes_path = 'nodes'
+    # Need to clean 'nodes' before each provision otherwise it uses existing data
+    if File.exist?(chef.nodes_path)
+      if File.directory?(chef.nodes_path)
+          Dir.foreach(chef.nodes_path) do |file|
+            if ((file.to_s != ".")\
+              and (file.to_s != "..")\
+              and (file.to_s != ".gitignore"))
```

Comment:
You could add additional files in here if you were actually testing with node data, though I haven't tested this myself because the chef-zero provisioner typically generates the data itself.